packages/vscode/package.json

```diff
@@ -2,6 +2,21 @@
   "name": "astro-vscode",
   "displayName": "Astro",
   "description": "Language support for Astro",
+  "categories": [
+    "Programming Languages",
+    "Formatters",
+    "Linters"
+  ],
+  "qna": false,
+  "keywords": [
+    "astro",
+    "node",
+    "typescript",
+    "javascript",
+    "vue",
+    "svelte",
+    "Static site generator"
+  ],
```

Comment:
All the changes here are inspired by what other extensions do. It might seems weird for the Astro extension to be in the "Formatters" category for example, but since it formats Astro files, it's considered as such